packages/fxa-auth-server/lib/senders/emails/templates/passwordResetAccountRecovery/en.ftl

```diff
@@ -1,9 +1,8 @@
-passwordResetAccountRecovery-subject-2 = Your password has been reset
-passwordResetAccountRecovery-title-2 = Password reset successfully
-# Details of the device and date/time that used account recovery key follow.
-passwordResetAccountRecovery-description-2 = You used your account recovery key to update your password from:
-# Text for button action to initiate creating new account recovery key
-passwordResetAccountRecovery-action-3 = Create account recovery key
-passwordResetAccountRecovery-regen-required-mjml-1 = You’ll need to sign in again on all of your synced devices. Remember to create a new account recovery key to replace the one you used.
-# After the colon, there's a link to https://accounts.firefox.com/settings/account_recovery
-passwordResetAccountRecovery-regen-required-txt-1 = You’ll need to sign in again on all of your synced devices. Remember to create a new account recovery key to replace the one you used:
+passwordResetAccountRecovery-subject-3 = Password updated
```

Comment:
Not a blocker, just a passing question: Is there a reason the subject uses `updated` when the rest of the email uses `reset`?

Reply:
I'd standardized to one of the two existing subject lines, but this wasn't in Figma so I'm not picky. Will switch to "Your password has been reset" for all these emails, and will leave "Password updated" for the password change flow from settings.